### PR TITLE
ISSUE-181: Support for Figma. Refactor Canva support.

### PR DESCRIPTION
**Pull request by few-bits**
### Pre-PR Checklist

- [ *] I have run `npm test` locally and there are no warnings or errors, from lint or test
- [ *] I have added screenshots if applicable

### Summary

Support for Figma

### Screenshots

CSS selector for Figma:
Screenshot from 2023-03-30 00-43-56

Extension setup:
Screenshot from 2023-03-30 00-52-42

### Issue

https://github.com/wakatime/chrome-wakatime/issues/181

## How Has This Been Tested?

* built and installed locally
* "https://www.figma.com/" added to white list
* open random project at Figma
* discover "projectDetails" variable at runtime
* Screenshot from 2023-03-30 01-39-45

### Reviewers

- @alanhamlett

### CODE REVIEW

File: src/wakatimeScript.ts [hunk 1]
```diff
@@ -243,21 +243,50 @@ const recordHeartbeat = async (apiKey: string, payload: Record<string, unknown>)
...
+
+const parseFigma = () => {
+  const projectName = (document.querySelector('span[data-testid="filename"]') as HTMLElement)
+    .innerText;
+  return {
+    editor: 'Figma',
+    language: 'Figma Design',
+    project: projectName,
+  };
+};
```
Comment (jvelezpo) [inline src/wakatimeScript.ts:267]: Thanks for your contribution.
Here we are missing a check to only send heartbeats when a user has a project open
something like [this](https://github.com/wakatime/chrome-wakatime/pull/185/files#diff-5fe6967e0cf99e4aad7de15da29ed8cf11e005156620e5a399cb0ee243b9fe66R260)

```
  const figmaProject = document.getElementsByClassName('gpu-view-content');
  if (figmaProject.length === 0) return;
```

**Comment by jvelezpo** [comment]
Please bump versions for chrome and FF under `src/manifests`, bumping patch should be good

File: src/wakatimeScript.ts (hunk 1 above)
Comment (few-bits) [reply src/wakatimeScript.ts]: Right, fixed. Thank you for review.

**Comment by few-bits** [comment]
> Please bump versions for chrome and FF under `src/manifests`, patch bumping should be good

done

**Comment by few-bits** [comment]
Fixed sendPostRequestToApi method. New Figma's project is on dashboard:
Screenshot from 2023-03-30 18-20-39

**Comment by few-bits** [comment]
@jvelezpo please, review again

**Review by jvelezpo — APPROVED** [review]

**Pull request merged**